javascript-ajax/users.js

```diff
@@ -0,0 +1,20 @@
+var $userList = document.querySelector('#user-list');
+
+function getUserData() {
+  var xhr = new XMLHttpRequest();
+  xhr.open('GET', 'https://jsonplaceholder.typicode.com/users');
+  xhr.responseType = 'JSON';
+  xhr.addEventListener('load', function () {
+    console.log('status:', xhr.status);
+    console.log('reponse:', xhr.response);
+    var parsedData = JSON.parse(xhr.response);
```

Comment:
This shouldn't be necessary. If your data wasn't parsed when you received it, it's probably because you set the response type to `"JSON"` rather than `"json"`